src/guides/v2.2/cloud/live/live.md

```diff
@@ -37,33 +33,51 @@ We strongly recommend testing in these environments due to the complexity of you
 
 ## Set up Magento Security Scan Tool {#security-scan}
 
-The Magento Security Scan Tool enables you to regularly monitor your store websites and receive updates for known security risks, malware, and out of date software. This is a free service available for all implementations and versions of {{site.data.var.ece}}. You access the tool through your [Magento Marketplace account](https://account.magento.com/customer/account/login).
+The Magento Security Scan Tool enables you to regularly monitor your store websites and receive updates for known security risks, malware, and out-of-date software. This is a free service available for all implementations and versions of {{site.data.var.ece}}. You access the tool through your [Magento Marketplace account](https://account.magento.com/customer/account/login).
 
 *  Monitor your sites security status and applied security updates
-*  Receive security updates and site specific notifications
+*  Receive security updates and site-specific notifications
 
-For detailed instructions to set up and perform scans, see the [Magento User Guide](http://docs.magento.com/m2/ee/user_guide/magento/security-scan.html). Typically, you want to start using this tool as you enter UAT testing.
+For detailed instructions to set up and perform scans, see the [Magento User Guide](http://docs.magento.com/m2/ee/user_guide/magento/security-scan.html). Typically, you want to start using this tool as you enter user acceptance testing (UAT).
 
-Each site to be scanned must be registered through Magento Security Scan tab. This registration process includes acceptance of Magento’s disclaimer prior to scanning. You control both scan scheduling and the authorization of personnel to be notified when each scan is completed. Scans can be scheduled for either a specific, recurring date and time or on-demand as required.
+Each site you scan must be registered through the Magento Security Scan tab. The registration process requires you to accept Magento’s disclaimer prior to scanning. You control both scan scheduling and the authorization of personnel to be notified when each scan is complete. You can schedule scans for a specific, recurring date and time or on demand as required.
 
+{%
+include note.html
+type='info'
+content='The security scan tool uses the following public IP addresses:
+
+```text
+52.72.230.169
+52.86.204.1
+52.87.98.44
+```
+
+You must whitelist these IP addresses in your network firewall rules to allow the tool to scan your site.'
+%}
+
+## Scan your site
+
+{:.procedure}
 To scan your site:
 
 1. Access your [Magento Marketplace account](https://account.magento.com/customer/account/login).
 1. Click the Security Scan tab and select **Go to Security Scan**.
-1. In the **Actions** column for the site, select Run Scan. A notification status displays the scheduled scan.
+1. In the **Actions** column for the site, select **Run Scan**. A notification status displays the scheduled scan.
```

Comment:
```suggestion
1. In the _Actions_ column for the site, select **Run Scan**. A notification status displays the scheduled scan.
```

We've been using _italics_ for location and **bold** for step action. Definitely not a show-stopper.